appcell-resources/js/common.js

```diff
@@ -840,6 +841,52 @@ cm.validateDescription = function(descriptionDetails, descriptionSpan) {
 	}
 	return isValidDescription;
 };
+// Validation Check
+cm.validateCellURL = function (cellURL, span) {
+    var isHttp = cellURL.substring(0, 5);
+    var isHttps = cellURL.substring(0, 6);
+    var minURLLength = cellURL.length;
```

Comment:
urlLength is defined in Line 852 already. No need to define minURLLength which has wrong meaning.